packages/unchained-client/generator/openapitools-dev.json

```diff
@@ -5,18 +5,18 @@
     "version": "5.4.0",
     "storageDir": "~/.openapitools",
     "generators": {
-      "ethereum": {
-        "inputSpec": "https://dev-api.ethereum.shapeshift.com/swagger.json",
+      "avalanche": {
+        "inputSpec": "https://dev-api.avalanche.shapeshift.com/swagger.json",
         "generatorName": "typescript-axios",
-        "output": "#{cwd}/src/generated/ethereum",
+        "output": "#{cwd}/src/generated/avalanche",
         "reservedWordsMappings": { "in": "in" },
         "additionalProperties": {
           "supportsES6": "true",
           "useSingleRequestParameter": true
         }
       },
       "bitcoin": {
-        "inputSpec": "https://dev-api.bitcoin.shapeshift.com/swagger.json",
+        "inputSpec": "http://api.bitcoin.localhost/swagger.json",
```

Comment:
fix

Reply:
addressed in https://github.com/shapeshift/lib/pull/914/commits/080c21f5f8f5aa8c96b55e31126c9b828da30d52